plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/query/index/HudiIndexSupport.java

```diff
@@ -28,4 +28,9 @@ Map<String, List<FileSlice>> lookupCandidateFilesInMetadataTable(
             TupleDomain<String> regularColumnPredicates);
 
     boolean canApply(TupleDomain<String> tupleDomain);
+
+    default boolean shouldKeepFileSlice(FileSlice slice)
```

Comment:
Change the name and flip the return value
```suggestion
    default boolean shouldSkipFileSlice(FileSlice slice)
```